staging/src/k8s.io/api/certificates/v1beta1/types.go

```diff
@@ -46,6 +46,7 @@ type CertificateSigningRequest struct {
 // Kubernetes and cannot be modified by users.
 type CertificateSigningRequestSpec struct {
 	// Base64-encoded PKCS#10 CSR data
```

Comment:
> // Base64-encoded PKCS#10 CSR data

isn't it already documented?

Reply:
I think it would be more clearer to mention that it is base64 encoded when specified in JSON or YAML, right?